frontend/core/blocks/mkv-to-mp4/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Use the Python 3.9 base image
+FROM python:3.9
+
+# Set the working directory
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+COPY . .
```

Comment:
You can replace this with `COPY requirements.txt .` because we only need the requirements.txt file and the computation.py file that gets added later in this file. We don't need the sample.mkv to be there.